pwiz_tools/Skyline/SkylineFiles.cs

```diff
@@ -596,7 +596,7 @@ private RCalcIrt FindIrtDatabase(IWin32Window parent, string documentPath, RCalc
                             var message = TextUtil.SpaceSeparate(
                                 Resources.SkylineWindow_FindIrtDatabase_The_database_file_specified_could_not_be_opened,
                                 e.Message);
-                            MessageBox.Show(message);
+                            MessageDlg.Show(this, message);

```

Comment:
I think "this" should be "parent".

Reply:
I still really think you need to use "parent" here and not "this". If "dlg.ShowDialog(parent)" is used above, then you need the message form to use the same parent.

Reply:
Thanks, fixed.

I think I have a similar issue in SettingsUI\EditIsotopeEnrichmentDlg.cs and SettingsUI\EditCEDlg.cs where we formerly had parentless message boxes. In both cases the new MessageDlg is currently parented to a datagrid but should probably be using the datagrid's parent, yes?